charts/eg/README.md

```diff
@@ -0,0 +1,26 @@
+# Usage
+
+[Helm](https://helm.sh) must be installed to use the charts.  Please refer to
+Helm's [documentation](https://helm.sh/docs) to get started.
+
+Once Helm has been set up correctly, add the repo as follows:
+
+``` shell
+  helm repo add eg https://gateway.envoyproxy.io/helm-charts
```

Comment:
since we are storing crds in the `crds/` directory, I would agree with https://github.com/envoyproxy/gateway/pull/651#issuecomment-1446383842 and make it very clear about managing CRDs separately from the chart and using `--skip-crds`

Reply:
Done. Thanks.